src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -107,12 +109,12 @@ public String getApiUri() {
         StandardCredentials credentials = Connector.lookupScanCredentials(observer.getContext(), apiUri, scanCredentialsId);
         GitHub github = Connector.connect(apiUri, credentials);
         if (credentials != null && !github.isCredentialValid()) {
-            listener.getLogger().format("Invalid scan credentials, skipping%n");
+            listener.getLogger().format("Invalid scan credentials %s to connect to %s, skipping%n", CredentialsNameProvider.name(credentials), apiUri == null ? "github.com" : apiUri);
             return;
         }
 
         if (!github.isAnonymous()) {
-            listener.getLogger().format("Connecting to GitHub using %s%n", CredentialsNameProvider.name(credentials));
+            listener.getLogger().format("Connecting to %s using %s%n", apiUri == null ? "github.com" : apiUri, CredentialsNameProvider.name(credentials));
```

Comment:
@cyrille-leclerc This only makes sense when you are working with a GHE instance. When you configure an instance, its endpoint has always the same format: `http(s)://hostname/api/v3/`. For that reason, the important part is the `hostname`.